#classification state
#ROSヘッダ
std_msgs/Header header
#位置
geometry_msgs/Point[] position